config/locales/en.yml

```diff
@@ -196,6 +196,8 @@ en:
       long: ! '%B %d, %Y %H:%M'
       short: ! '%d %b %H:%M'
     pm: pm
+  placeholder:
+    full_name_or_nickname_or_event_name: "Full Name or Nickname or Event Name"
```

Comment:
I wonder why the build is failing. I ran Travis-CI on my own fork and seems to be building successfully(as shown [here](https://travis-ci.org/krngrvr09/travel-support-program)).

Also the tests which are failing have nothing to do with the Search functionality. How do you think I should debug this?